CONTRIBUTING.md

```diff
@@ -11,8 +11,8 @@ In this document:
 
 ## Adding content ##
 
-Before adding content, submit an issue with a suggestion for your proposed
-article. Provide detail on what the article would discuss, and how it would
+Before adding content or editing content, submit an issue with a suggestion for your proposed
```

Comment:
I think it sounds better if we say "Before adding or editing content" instead of repeating content twice